framework/src/controls/PIDTransientControl.C

```diff
@@ -0,0 +1,148 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// MOOSE includes
+#include "PIDTransientControl.h"
+#include "Function.h"
+#include "Transient.h"
+
+registerMooseObject("MooseApp", PIDTransientControl);
+
+InputParameters
+PIDTransientControl::validParams()
+{
+  InputParameters params = Control::validParams();
+  params.addClassDescription(
+      "Sets the value of a 'Real' input parameter (or postprocessor) based on a Proportional "
+      "Integral "
+      "Derivative control of a postprocessor to match a target a target value.");
```

Comment:
Reflow this. Looks like a bad `clang-format` side effect of formatting already broken up strings.